File: holoken-core/src/test/java/com/holokenmod/creation/RandomizerMock.java
package com.holokenmod.creation;

import com.holokenmod.Randomizer;

public class RandomizerMock implements Randomizer {
	@Override
	public void discard() {
		//nothing to do
	}
	
	@Override
	public int nextInt(int maximumNumber) {
		return 0;
	}
	
	@Override
	public double nextDouble() {
		return 0;
	}
}

File: holoken-core/src/test/java/com/holokenmod/creation/RandomizerMock.kt
package com.holokenmod.creation

import com.holokenmod.Randomizer

class RandomizerMock : Randomizer {
    override fun discard() {
        //nothing to do
    }

    override fun nextInt(maximumNumber: Int): Int {
        return 0
    }

    override fun nextDouble(): Double {
        return 0.0
    }
}